kodilla-stream/src/test/java/com/kodilla/stream/portfolio/BoardTestSuite.java

```diff
@@ -132,6 +133,33 @@ public void testAddTaskListFindLongTasks() {
         //Then
         Assert.assertEquals(2, longTasks);
     }
+    @Test
+    public void testAddTaskListAverageWorkingOnTask() {
+        //Given
+        Board project = prepareTestData();
 
+        //When
+        List<TaskList> inProgressTasks = new ArrayList<>();
+        inProgressTasks.add(new TaskList("In progress"));
 
+        long taskQuantity = project.getTaskLists().stream()
+                .filter(inProgressTasks::contains)
+                .flatMap(t -> t.getTasks().stream())
+                .map(Task::getCreated)
+                .map(t1 -> t1.until(LocalDate.now()))
+                .map(Period::getDays)
+                .filter(t2 -> t2 > 0)
```

Comment:
Czy ten warunek jest potrzebny? Wydaje mi się ze wystarczyłoby count po flatmap

Reply:
Jest to "wyjście przed szereg", ale uznałem, że wliczanie do średniej tasków stworzonych "dzisiaj" znacząco zakłamuje obraz średniego czasu wykonywania zadania.